src/main/java/seedu/address/logic/commands/ImportCommand.java

```diff
@@ -5,42 +5,40 @@
 import java.io.FileReader;
 import java.io.IOException;
 import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
 import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.stream.Stream;
 
 import com.opencsv.CSVReader;
 import com.opencsv.exceptions.CsvValidationException;
 
 import seedu.address.logic.commands.exceptions.CommandException;
-import seedu.address.logic.parser.ParserUtil;
-import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.logic.parser.CsvPersonParser;
+import seedu.address.logic.parser.CsvRowParser;
 import seedu.address.model.Model;
-import seedu.address.model.assignment.Assignment;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Github;
-import seedu.address.model.person.Name;
 import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
-import seedu.address.model.person.Telegram;
-import seedu.address.model.tag.Tag;
+import seedu.address.model.person.exceptions.DuplicatePersonException;
 
 /**
- * Import command to load data from specified CSV file location.
+ * Command to import person data from a CSV file into the address book.
+ * Expects a CSV file with a specific header format that is case-insensitive but order-sensitive.
  */
 public class ImportCommand extends Command {
 
     public static final String COMMAND_WORD = "import";
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Imports person data from a CSV file.\n"
-            + "Parameters: FILE_PATH"
-            + "[" + PREFIX_PATH + "FILE_PATH]\n"
-            + "Example: " + COMMAND_WORD + " " + PREFIX_PATH + "data/persons.csv";
+        + "Parameters: FILE_PATH"
+        + "[" + PREFIX_PATH + "FILE_PATH]\n"
+        + "Example: " + COMMAND_WORD + " " + PREFIX_PATH + "data/persons.csv";
+    public static final String CORRECT_HEADER_USAGE =
+        "Your header should be Name, Phone, Email, Telegram, Tags, Github, Assignments, WeeksPresent"
```

Comment:
I feel like we should be less directed towards the user and more like returning a message such as "The header of the csv file should be Name, Phone, Email, Telegram, Tags, Github, Assignments, WeeksPresent"